financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerViewModel.kt

```diff
@@ -82,6 +84,16 @@ internal class AccountPickerViewModel @AssistedInject constructor(
         )
     }
 
+    private fun loadInstitution() {
+        suspend {
+            val sync = getOrFetchSync()
+            val manifest = sync.manifest
+            requireNotNull(manifest.activeInstitution)
+        }.execute {
+            copy(institution = it)
+        }
+    }
```

Comment:
kind of related - this could benefit from the work [here](https://github.com/stripe/stripe-android/pull/8317) by attempting to re-sync if the institution is missing